.sync.yml

```diff
@@ -8,6 +8,40 @@
 .travis.yml:
   unmanaged: true
 
+# Requires the bundler_args to be supported.
```

Comment:
Why add this now if it can't be used?

Reply:
Saving myself (or someone else) future work.  That's the only thing missing.  PDK adds that by default.